test/unit/util.test.js

```diff
@@ -2957,3 +2957,72 @@ describe('findCommonSubpath method', function () {
   });
 
 });
+
+describe('getAuthHelper method - OAuth2 Flows', function() {
```

Comment:
@SahilChoudhary22 Can we add more tests that checks for behaviour of all fields that can be added as part of OAuth2 auth in postman? Here we're only having `accessTokenUrl` and `grant_type`.